examples/connext_dds/compression/c++/USER_QOS_PROFILES.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<!-- 
+Description
```

Comment:
Leave just the copyright notice and bump the year to 2021

Reply:
done